server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -24,6 +24,8 @@ export function handleResponse<T>(
         (result) => {
             if (cache) {
                 response.setHeader("Cache-Control", "public, max-age=31536000");
+            } else {
+                response.setHeader("Cache-Control", "no-store, max-age=0");
```

Comment:
Does "no-store" tells the browser to not cache? Or browser would still cache?

Reply:
It will not cache, and adding "max-age=0" as well forces the cache to revalidate (clears the cache) to prevent the cache using a stale value (that was the recommendation I found [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#preventing_caching)).